wsl-pro-service/debian/rules

```diff
@@ -14,6 +14,11 @@ export GOTOOLCHAIN := local
 	dh $@ --builddirectory=_build --with=apport
 
 execute_after_dh_auto_clean:
+	# Overriding version file
+	# We must do this during the source package build because we still have access to git.
+	# We can't do this during the binary package, so we cannot use the usual -ldflags.
+	[ "$(shell cat ../common/version)" = "Dev" ] && go run ../tools/build/compute_version.go > ../common/version || true
```

Comment:
That would not work right? because we will not have `../` anymore from the source package? It needs to be inside the source package itself (so under that directory).

Reply:
I would argue for the root of `wsl-pro-service`, so that it’s clear it’s the file that is generated from the project.